problem1/himanshu.rb

```diff
@@ -0,0 +1,2 @@
+class Abc
```

Comment:
@HimanshuTiwariPlum : What is the meaning of `Abc` here ? Please elaborate.

Reply:
@AjayBarot changed the name of the class and make it appropriate according to Exercise and also make input from the user at runtime. Please take fetch for latest commit.